pyproject.toml

```diff
@@ -195,7 +195,6 @@ select = [
   "PT009",   # pytest-unittest-assertion
   "PT010",   # pytest-raises-without-exception
   "PT011",   # pytest-raises-too-broad
-  "PT012",   # pytest-raises-with-multiple-statements
```

Comment:
left a comment on this here: https://github.com/mlflow/mlflow/pull/12658#discussion_r1680706603